pipelines/SpeciesTreeFromBusco/main.nf

```diff
@@ -252,7 +252,7 @@ process runGffread {
     script:
     """
     mkdir -p cdna
-    ${params.gffread_exe} -w cdna/$genome -g $genome $busco_annot
+    ${params.gffread_exe} -x cdna/$genome -g $genome $busco_annot
```

Comment:
As discussed in person, this was changed to write a file with the CDS instead of the spliced exons, as only the CDS is needed.